src/pages/LinkCompanyPage/hooks.ts

```diff
@@ -0,0 +1,31 @@
+import { useQueryWithClient } from "@deskpro/app-sdk";
+import { getAccountService, searchCompaniesService } from "@/services/copper";
+import { QueryKey } from "@/query";
+import type { Maybe } from "@/types";
+import type { Company, Account } from "@/services/copper/types";
+
+export type Result = {
+  isLoading: boolean;
+  companies: Company[];
+  account: Maybe<Account>;
+};
+
+type UseSearchOrg = (q?: string) => Result;
+
+const useSearchOrg: UseSearchOrg = (q) => {
+  const account = useQueryWithClient([QueryKey.ACCOUNT], getAccountService);
+
+  const companies = useQueryWithClient(
+    [QueryKey.COMPANIES, q as string],
```

Comment:
This typecast is inaccurate, the input of `q` (which we should rename to `query` or something else) is `string|undefined`. Probably best to change the input type so that it's always a `string` instead.

Reply:
fixed the type